commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/service/PageTabSerializationService.kt

```diff
@@ -33,7 +32,7 @@ internal class PageTabSerializationService(
         fileListSerializer.deserializeFileList(serializer.deserializeSubmission(content, format), source)
 
     override fun deserializeSubmission(file: File): Submission =
-        when (fromFile(file)) {
+        when (SubFormat.fromFile(file)) {
```

Comment:
![but-why](https://user-images.githubusercontent.com/19352419/111144235-bab91f80-857e-11eb-83c0-95b8b0299261.gif)

Reply:
I founded  hard while coding to identify what was function doing it without the class name like figured out was that fromFile from so I added back, another option I think will be name function `SubFormatFromFile`